### PR TITLE
update README file

### PR DESCRIPTION
**Pull request by ANSHIKA1806**
@Surajv311 
Sir, 
I've modified the link, added some more description and aligned the name "Jssconnect" in center.
Please accept my pull request and let me know if any further changes you expect.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
-https://jssconnect.herokuapp.com/
+[Visit our website here](https://jssconnect.herokuapp.com/)
+<div align="center">
 <h2 style="text-align:center;font-size:53px;color:red"><span style="color:blue">JSS</span>CONNECT</h2>
+</div>
 
 <h3 style="text-align:center;color:yellow;font-size:1.8rem;">One Destination for meeting all your educational needs related to college study.</h3>
 <p style="text-align:center;font-size:1.5rem;">Jss Connect is an e-platform for helping engineering students in their academics.</p>
+<p style="text-align:center;font-size:1.5rem;">
+It has all the stuff to satisfy the hunger of today’s engineers.</p>
```
Comment (Surajv311) [inline README.md:9]: This line isn't required, kindly remove it.

File: README.md [hunk 2]
```diff
@@ -142,11 +146,12 @@ use db <db name>
 show collections
 <db name> .find()
 ```
-<h3>***Note</h3>
+<h3>*** Note ***</h3>
 *Get connection string from Mongo Atlas by creating a cluster or you can also use your locally installed mongodb<br>
```
Comment (Surajv311) [inline README.md]: Make all these points in a bulleted list.

**Review by Surajv311 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Surajv311) [inline README.md]: This line isn't required, kindly remove it.

**Comment by ANSHIKA1806** [comment]
@Surajv311 
Sir, I've changed the README file according to you.
Please review it and accept my pull request.

**Comment by Surajv311** [comment]
LGTM

**Pull request merged**